public/components/marketing-testimonials/9.html

```diff
@@ -0,0 +1,56 @@
+<div class="my-0 mx-auto flex flex-wrap items-center justify-center">
+
+    <!-- Card 1 -->
+    <section class="w-80 rounded-sm shadow-lg bg-slate-900 px-10 py-8 text-white m-2 ">
+        <div class="flex flex-col items-center text-center">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="52" height="52" class="my-3">
+                <path fill="none" d="M0 0h24v24H0z" />
+                <path fill="white"
+                    d="M4.583 17.321C3.553 16.227 3 15 3 13.011c0-3.5 2.457-6.637 6.03-8.188l.893 1.378c-3.335 1.804-3.987 4.145-4.247 5.621.537-.278 1.24-.375 1.929-.311 1.804.167 3.226 1.648 3.226 3.489a3.5 3.5 0 0 1-3.5 3.5c-1.073 0-2.099-.49-2.748-1.179zm10 0C13.553 16.227 13 15 13 13.011c0-3.5 2.457-6.637 6.03-8.188l.893 1.378c-3.335 1.804-3.987 4.145-4.247 5.621.537-.278 1.24-.375 1.929-.311 1.804.167 3.226 1.648 3.226 3.489a3.5 3.5 0 0 1-3.5 3.5c-1.073 0-2.099-.49-2.748-1.179z" />
+            </svg>
+            <p class="my-4 font-normal tracking-wide">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Corporis
```

Comment:
Should use `space-y-4` on the parent instead of `my-4` here and `font-normal` isn't needed.